terraform/versions.tf

```diff
@@ -1,3 +1,8 @@
 terraform {
-  required_version = "~> 0.12.9"
-}
\ No newline at end of file
+  required_version = ">= 0.13"
```

Comment:
the pr desc mentions tf 0.14 but 0.13 is pinned here. any reason?

Reply:
Whoops, I missed that, but the automatic migration tool nor the validator did not complain: that's why it slipped. I can fix this in this PR or in a separate one. However you prefer.